src/current/v23.2/protect-changefeed-data.md

```diff
@@ -0,0 +1,81 @@
+
+However, if the changefeed lags too far behind, the protected changes could lead to an accumulation of garbage. This could result in increased disk usage and degraded performance for some workloads.
+
+## Prevent garbage accumulation
+
+To prevent an accumulation of protected changes that could impact performance, consider defining an expiration duration:
+
+- [`changefeed.protect_timestamp.max_age`](#changefeed-protect_timestamp-max_age): a [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) to define a protected timestamp expiration for all changefeeds on a cluster.
+- [`gc_protect_expires_after`](#gc_protect_expires_after): a [changefeed option]({% link {{ page.version.version }}/create-changefeed.md %}#options) to define a protected timestamp expiration for a changefeed.
+
+In general, a few hours to a few days are appropriate values for these settings. A lower protected timestamp expiration should not have adverse effects on your changefeed as long as the changefeed is running. However, if the changefeed pauses, you will need to [resume]({% link {{ page.version.version }}/resume-job.md %}) it before the defined expiration time. The value of this setting should reflect how much time the changefeed may remain paused before it is canceled.
+
+### `changefeed.protect_timestamp.max_age`
+
+{% include_cached new-in.html version="v23.2" %} By default, the `changefeed.protect_timestamp.max_age` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) sets the maximum time that changefeeds making no forward progress will hold protected timestamp records. Once the `changefeed.protect_timestamp.max_age` duration is reached, the changefeed will fail with a permanent error. As a result, it is **critical to monitor for changefeed failures** because changefeeds will eventually fail with an unrecoverable error if they cannot progress before the duration is reached.
+
+This cluster setting is enabled by default to 4 days. To disable expiration of protected timestamp records, you can set `changefeed.protect_timestamp.max_age` to `0`; however, Cockroach Labs recommends implementing an expiration.
+
+`changefeed.protect_timestamp.max_age` is a cluster-wide setting affecting all changefeeds.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING changefeed.protect_timestamp.max_age = '120h';
+~~~
+
+{{site.data.alerts.callout_info}}
+`changefeed.protect_timestamp.max_age` applies only to **newly created changefeeds in v23.2**.
+
+If you are [upgrading to v23.2]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}), we recommend setting `protect_data_from_gc_on_pause` on any existing changefeeds to ensure that it does not enter a situation of infinite retries, which could prevent garbage collection. You can use the [`ALTER CHANGEFEED`]({% link {{ page.version.version }}/alter-changefeed.md %}) statement to add `protect_data_from_gc_on_pause` to existing changefeeds.
```

Comment:
Would you be able to add a link on the first occurrence of `protect_data_from_gc_on_pause` to https://www.cockroachlabs.com/docs/v23.1/create-changefeed#protect-pause?

Reply:
Added!